relay-general/src/pii/attachments.rs

```diff
@@ -1,7 +1,12 @@
 use std::borrow::Cow;
+use std::iter::FusedIterator;
+use std::str::Utf8Error;
 
+use encoding::all::UTF_16LE;
+use encoding::{Encoding, RawDecoder};
 use regex::bytes::RegexBuilder as BytesRegexBuilder;
-use regex::Regex;
+use regex::{Match, Regex};
+use relay_wstring::{Utf16Error, WStr};
```

Comment:
_nit_: Please move this import into its own group. We usually separate imports of workspace crates from external crates.